components/automate-ui/src/app/ui.component.ts

```diff
@@ -13,7 +13,8 @@ import { GetAllUserPerms } from './entities/userperms/userperms.actions';
 @Component({
   selector: 'app-ui',
   templateUrl: './ui.component.html',
-  styleUrls: ['./ui.component.scss']
+  styleUrls: ['./ui.component.scss'],
+  encapsulation: ViewEncapsulation.None,
```

Comment:
what does `ViewEncapsulation` do?

Reply:
@bcmdarroch It removes the style encapsulation so we can select and style the material component.
https://angular.io/api/core/ViewEncapsulation